doc/move_group_interface/move_group_interface_tutorial.rst

```diff
@@ -38,8 +38,10 @@ See the `YouTube video <https://youtu.be/_5siHkFQPBQ>`_ at the top of this tutor
 
  6. The robot moves its arm to the pose goal, avoiding collision with the box.
  7. The object is attached to the wrist (its color will change to purple/orange/green).
- 8. The object is detached from the wrist (its color will change back to green).
- 9. The object is removed from the environment.
+ 8. The robot moves its arm to a pose goal with orientation tolerance, avoiding collision with the box.
+ 9. The robot moves its arm to a pose goal with position tolerance, avoiding collision with the box.
+ 10. The object is detached from the wrist (its color will change back to green).
+ 11. The object is removed from the environment.
```

Comment:
If there is a video of the new demo, it would be great to update the youtube video demo link to show this new full process.

Reply:
could you suggest an updated tutorial steps numbered list if you see some improvement possible?

Reply:
I made a suggestion below to move your new steps after step 6.

Reply:
I chose to detach the object before planning with goal tolerance. Then the motion with and without object attached is kept one after the other.